.clang-format

```diff
@@ -1,6 +1,6 @@
 Language: Cpp
 BasedOnStyle: Google
-ColumnLimit: 120
+ColumnLimit: 100
```

Comment:
I'm fine with it but:
- I don't want to have more-or-less arbitrary style changes based on personal preferences.
- This should preferably be a separate PR.

Reply:
Okay!